### ISSUE TITLE
[Bug] Small bugs in Settings>Appearance page

### ISSUE THREAD

**Issue by MayurSMahajan** [AppFlowy-IO/AppFlowy#3429]
### Bug Description

The UI for the Appearance page in the settings looks like this for the latest branch:

Two bugs with the current UI:

1. Only the Theme value button changes the cursor to grab and shows a background color upon hover. All the other options, that is Theme, Font Family, Layout Direction, Default Text Direction do not get a grab cursor and background color upon hover.

2. In the Theme option, observe how there is a file icon followed by the text button followed by the reset icon
It would be better in terms of UX if the order was changed to put the file icon after the Text Button.

### How to Reproduce

1. Open AppFlowy
2. Go to Settings
3. Go to the Appearance tab 
4. Observe that hovering on Theme, Font Family, Layout Direction, Default Text Direction does not show a hover background and grab cursor.
5. Observe how the ordering of the Theme actions is not intuitive.

### Expected Behavior

1. Open AppFlowy
2. Go to Settings
3. Go to the Appearance tab
4. Hovering on all text buttons should show a background and grab cursor.
5. The Theme actions should be arranged as: Theme Text Button, Folder icon for Theme Picker, Reset Button

### Operating System

All

### AppFlowy Version(s)

latest main

### Screenshots

![image](https://github.com/AppFlowy-IO/AppFlowy/assets/47064215/9816f909-2e00-4d4f-b02b-8f56e8037d8d)

### Additional Context

So to sum up, to solve this issue we need to complete the following sub-tasks:

 - [ ] Make cursor grab and background color when hovering on Appearance Options Buttons.
 - [ ] Rearrange the Theme Picker Actions to: Theme Text Button, Folder icon for Theme Picker, Reset Button

**Comment by Nitin-Poojary** [comment]
Is this what you want?
![image](https://github.com/AppFlowy-IO/AppFlowy/assets/70025277/a41118e3-7b9f-4723-aadd-c726e95a4da8)

**Comment by MayurSMahajan** [comment]
Yes.
